src/Transports/MassTransit.Azure.ServiceBus.Core/MassTransit.Azure.ServiceBus.Core.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.3.0" />
+    <PackageReference Include="Azure.Identity" Version="1.6.0" />
```

Comment:
This seemed to be required otherwise i was still getting 401s. There's some inconsistency across the rest of the projects on this dependency

Reply:
And broke the build.

Reply:
![image](https://user-images.githubusercontent.com/3485935/163180694-f33bcaba-8972-46c3-8268-2ecaacf4baba.png)
Will align web jobs.
I'm thinking i leave storage for now - will see what comes out. I may end up back in storage anyway as i'm just doing a bit of a general manged identity pass

Reply:
I'm fine updated all the projects to the latest of the identity package, if that's actually required. I don't like updating beyond minimum as it often breaks people upgrading.